src/api/db/migrate/20240119114738_change_events_payload_from_text_to_mediumtext.rb

```diff
@@ -0,0 +1,11 @@
+class ChangeEventsPayloadFromTextToMediumtext < ActiveRecord::Migration[7.0]
+  def self.up
+    safety_assured { change_column :events, :payload, :text, limit: 16.megabytes - 1 }
```

Comment:
I think this triggered a [Strong Migrations](https://github.com/ankane/strong_migrations) warning, right? We may need to run this migration during [our maintenance window](https://github.com/openSUSE/ansible-obs#deployment-with-migration-with-downtime)

Reply:
yeah